packaging/centos63/cloud.spec

```diff
@@ -454,8 +454,14 @@ if [ -f "%{_sysconfdir}/cloud.rpmsave/management/db.properties" ]; then
     mv %{_sysconfdir}/cloud.rpmsave/management/db.properties %{_sysconfdir}/cloud.rpmsave/management/db.properties.rpmsave
 fi
 
+if [ -f %{_sysconfdir}/sysconfig/%{name}-management ] ; then
+    rm -f %{_sysconfdir}/sysconfig/%{name}-management
+fi
+
 chown -R cloud:cloud /var/log/cloudstack/management
 
+systemctl daemon-reload
```

Comment:
@ernjvr remove the systemctl thing, CentOS6/el6 does not have systemd. No need to reload anything here. Rest is LGTM.